src/components/ConsoleOutput.jsx

```diff
@@ -2,9 +2,18 @@ import isNil from 'lodash-es/isNil';
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import Inspector from 'react-inspector';
```

Comment:
How big is the footprint of `react-inspector`? Generally I try to avoid loading any library dependencies in the main bundle that aren’t needed to bootstrap a fresh Popcode environment. Since this is a UI library and asynchronously rendering React components is kind of a pain, I’m down to do it in a fast-follow if the library itself is small, although [this (sadly unmerged) PR](https://github.com/popcodeorg/popcode/pull/1426/files#diff-038c11f73da9e0721eb03924327efa1cR1) has some very nice prior art on async loading a component so as to async render a library dependency!